DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -31,5 +33,43 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                 new { customisationId }
             );
         }
+
+        public int GetProgressId(int candidateId, int customisationId)
+        {
+            return connection.QueryFirstOrDefault<int>(

```

Comment:
So currently this will return 0 as the progress id if there's no progress record which is not what we want. However HEEDLS-202 is going to handle this. Worth just leaving a TODO comment here saying something like:
```
// TODO HEEDLS-202: change QueryFirstOrDefault to creating progress record if not found
```
Just so we don't forget 🙂